README.md

```diff
@@ -356,9 +377,9 @@ The returned theme object can then be passed to the `Deck` tag via the `theme` p
 
 ## FAQ
 
-_How can I easily style the base components for my presentation?_
+**_How can I easily style the base components for my presentation?_**
 
-Historically, custom styling in Spectacle has meant screwing with a theme file, or using gross `!important` overrides. We fixed that. Spectacle is now driven by [emotion](https://github.com/emotion-js/emotion), so you can bring your own styling library, whether its emotion itself, or something like styled-components or glamorous. For example, if you want to create a custom Heading style:
+Historically, custom styling in Spectacle has meant screwing with a theme file, or using `!important` overrides. We fixed that. Spectacle is now driven by [emotion](https://github.com/emotion-js/emotion), so you can bring your own styling library, whether its emotion itself, or something like styled-components or glamorous. For example, if you want to create a custom Heading style:
```

Comment:
TINY nit, but `whether its emotion itself` should be `whether it's emotion itself`